server/database/models/manufacturers.js

```diff
@@ -0,0 +1,50 @@
+export function getAttributes(sequelize, DataTypes) {
+  return {
+    id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      primaryKey: true,
+      autoIncrement: true
+    },
+    name: {
+      type: DataTypes.TEXT,
+      allowNull: false
+    },
+    origin_country: {
```

Comment:
use field, 

so it will be 

```
originCountry: {
  field: 'origin_coutry', 
  type: DataTypes.TEXT,
   allowNull: false
}
```

this way you won't ever need snake_case when interacting with the models from the application code. 

Do this for all the columns, you can look at examples here: https://github.com/wednesday-solutions/node-express-graphql-template/blob/develop/server/database/models/products.js#L3-L37